keyboards/charue/sunsetter/readme.md

```diff
@@ -0,0 +1,15 @@
+# Sunsetter
+
+![Sunsetter](https://cdn.shopify.com/s/files/1/0476/6590/5825/products/Sunsetter-White_RedSun-0001_1024x1024@2x.png)
+
+A compact 40% (12x4) ortholinear keyboard kit made and sold by OLKB and Massdrop. [More info on qmk.fm](http://qmk.fm/planck/)
+
+* Keyboard Maintainer: [Coarse](https://github.com/coarse)
+* Hardware Supported: [Sunsetter PCB](https://charue-design.com/collections/sunsetter/products/sunsetter-pcb-extra)
+* Hardware Availability: [Charue Design](https://charue-design.com/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader to flash new firmware, here?

Reply:
Added instructions on jumping to bootloader using bootmagic or with the reset button